content/blog/2018/07/2018-07-03-remoting-over-message-bus-alpha-release.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "GSoC Project Update: Alpha release of remoting-kafka-plugin"
+tags:
+- plugins
+- gsoc
+- gsoc2018
+- remoting
+- alpha-release
+author: pvtuan10
+---
+
+=== Overview
+https://github.com/jenkinsci/remoting-kafka-plugin[Remoting Kafka Plugin] is a plugin developed under https://jenkins.io/projects/gsoc/[Jenkins Google Summer of Code 2018]. The plugin allows users to connect agents to Jenkins master using Kafka to solve the problem of current direct TCP communication between master and agent. A quick introduction of the project could be found in this  https://jenkins.io/blog/2018/06/18/remoting-over-message-bus/[introduction blogpost].
```

Comment:
A few changes for grammar/readability:: 
* "connect agents to Jenkins master" to "connect agents to a Jenkins master"
* "solve the problem of current direct TCP communication between master and agent" to "avoid direct TCP communication between master and agent"
* "project could be found" to "project can be found"